data/help.txt

```diff
@@ -74,12 +74,22 @@ help "navigation"
 	`When you reach a new system, you can press <Land on planet / station> to land on any inhabited planets that are there.`
 	`Also, don't worry about crashing into asteroids or other ships; your ship will fly safely below or above them.`
 
-help "outfitter"
+help "outfitter 1"
```

Comment:
This change will make the this particular dialog display again, even for players that have seen this text before.

Reply:
I personally don't see a problem with that and find it more standardized going forward, but I can take it off if that's what people prefer.